apps/web-giddh/src/app/shared/header/header.component.scss

```diff
@@ -766,6 +766,79 @@ ul.sub-list {
     left: 15px;
     border: 1px solid $gainsboro;
 }
+.switch-branch-wrapper{
+    .btn-group{
+        display: flex;
+        a.dropdown-toggle{
+            font-size: 16px;
+            display: flex;
+            padding: 15px;
+            font-family: 'opensans-bold';
+            color: $nero;
+            border-bottom: 1px solid $gainsboro;
+            width: 100%;
+            align-items: center;
+            justify-content: space-between;
+            word-break: break-word;
+            white-space: inherit;
+            span{
+                font-size: 9px;
+                float: right;
+                padding-top: 4px;
+                transform: rotate(270deg);
+                border: 2px solid $gray;;
+                padding: 3px;
+                border-radius: 50%;
+                justify-content: center;
+                align-items: center;
+                display: flex;
+                color: $gray;
+            }
+        }
+        .dropdown-menu{
+            left: auto !important;
+            right: -270px !important;
+            top: 10px !important;
+            transform: translateY(0px);
+            bottom: auto;
+            border: 1px solid #d9d9d9;
```

Comment:
use color var

Reply:
remove code, that is architecture css which should not go to stage

Reply:
remove code, that is architecture css which should not go to stage